crates/pop-cli/src/commands/test/create_snapshot.rs

```diff
@@ -202,25 +166,25 @@ mod tests {
 	#[test]
 	fn display_works() {
 		let mut command = TestCreateSnapshotCommand::default();
-		command.from.uri = Some(DEFAULT_REMOTE_NODE_URL.to_string());
+		command.from.uri = Some("ws://localhost:9944".to_string());
```

Comment:
Worth declaring a constant similar to `DEFAULT_REMOTE_NODE_URL` but for the local instance ?